tools/coverity/misra.config

```diff
@@ -18,6 +18,14 @@
             deviation: "Directive 4.9",
             reason: "Allow inclusion of function like macros. Logging is done using function like macros."
         },
+        {
+            deviation: "Directive 4.12",
+            reason: "Allow use of malloc. This library uses malloc to create cryptographic objects."
```

Comment:
I would like to not use malloc, is this mandated by the PKCS11 interface or is it possible to allocate these statically? I am skeptical that malloc is required for PKCS11 to work, seems hard to believe they would have made such a fundamental mistake.

Reply:
Malloc is used by the underlying mbed TLS library as well as by our implementation. It is not mandated, and we could switch to a statically allocated memory pool to use for operations